src/app/(content-pages)/transform-property/TransformPropertyPage.tsx

```diff
@@ -1,20 +1,15 @@
-import { Accordion, AccordionItem } from "@nextui-org/react";
 import Image from "next/image";
 import beforeAfter from "@/images/beforeAfter.png";
+import { ArrowUpRight } from "@phosphor-icons/react";
 
 export default function TransformPropertyPage() {
   return (
     <div className="flex flex-col min-h-screen">
       <div className="flew-grow container mx-auto pt-20">
         <h1 className="heading-3xl font-bold mb-6">Transform a Property</h1>
-
-        <p className="body-md">
-          Clean & Green Philly aims to help reduce gun violence by intervening
```

Comment:
not a breaking issue, but when writing copy, it's always best to replace usages of certain symbols like `&` with `&amp;` — you can read more about HTML entities here: https://developer.mozilla.org/en-US/docs/Glossary/Entity